python_modules/libraries/dagster-dbt/dagster_dbt/cloud/asset_defs.py

```diff
@@ -325,6 +329,24 @@ def _assets(context: OpExecutionContext):
 
                 dbt_options.append(f"--vars '{json.dumps(dbt_vars)}'")
 
+            # Map the selected outputs to dbt models that should be materialized.
+            #
+            # HACK: This selection filter works even if an existing `--select` is specified in the
+            # dbt Cloud job. We take advantage of the fact that the last `--select` will be used.
+            #
+            # This is not ideal, as the triggered run for the dbt Cloud job will still have both
+            # `--select` options when displayed in the UI, but parsing the command line argument
+            # to remove the initial select using argparse.
```

Comment:
Just wanted to call this out this is definitely unideal, but I don't want to spend an eternity understanding the semantics of converting a between the string representation of a command line argument and its python model representation. 

Surprisingly, I couldn't find much information about this on `click`/`argparse`.

Reply:
lolol well seems ok for now